spec/lib/theme_javascript_compiler_spec.rb

```diff
@@ -83,7 +83,7 @@ def statement(template)
     end
 
     # might change/break when updating ember
-    EMBER_INTERNAL_ID = 29
+    EMBER_INTERNAL_ID = 28
```

Comment:
Could this use the `Ember::VERSION` or `BASE_COMPILER_VERSION` instead?

Reply:
No I don't think so, this is some thing in Ember compiler, it's not something "to bump" at each version.

Reply:
Ah I understand. My recommendation then would be to ignore that value in the test.

Reply:
A bit more background, that's a glimmer vm opcode for helper (see: https://github.com/glimmerjs/glimmer-vm/blob/master/packages/@glimmer/wire-format/lib/opcodes.ts#L35) that has changed after this commit: https://github.com/glimmerjs/glimmer-vm/commit/b4707985fe462052454e15ca5a537b3103156fdd

I've changed the test so it doesn't hard-code this value and updated the names so they're a bit more self-explanatory.